cli/state.go

```diff
@@ -776,7 +776,7 @@ var StateGetActorCmd = &cli.Command{
 		fmt.Printf("Nonce:\t\t%d\n", a.Nonce)
 		fmt.Printf("Code:\t\t%s (%s)\n", a.Code, strtype)
 		fmt.Printf("Head:\t\t%s\n", a.Head)
-		fmt.Printf("Predictable address:\t\t%s\n", a.Address)
+		fmt.Printf("Deterministic address:\t\t%s\n", a.Address)
```

Comment:
I think this should stay as `Predictable`

Reply:
Nope, we moved away from that term in favour of Delegated.

Reply:
I used Deterministic because I wrongly thought this could also carry an f1 or f3 address, in addition to the f4.